apps/balancer-tools/src/app/apr/(utils)/tokenApr.ts

```diff
@@ -55,14 +62,14 @@ export async function getPoolTokensAprForDateRange(
             return {
               address: tokenAddress,
               symbol,
-              yield: await getAPRFromRateProviderInterval(
-                rateProviderAddress as Address,
-                startAt,
-                endAt,
+              yield: await getAPRFromRateProviderInterval({
```

Comment:
I really don't care about it all that much, but if we're going to change one signature to be an object we might as well change all of them to ensure the same pattern. Personally I think this makes the code more verbose but I'm ok with that as long as we're consistent/there's a reason for each exception. wdyt?

Reply:
Agree! During development, I just like to make sure I'm not getting errors because I swap the props order xD but will keep it consistent 🫡